api/apps/articles/model/index.coffee

```diff
@@ -118,11 +118,19 @@ Q = require 'bluebird-q'
   }
 
 @present = (article) =>
+  if article?.scheduled_publish_at
+    scheduled = moment(article.scheduled_publish_at).toISOString()
+  else
+    scheduled = null
```

Comment:
Ah ok, so we should also do this for `published_at` too, right? Since drafts don't have `published_at`

Could be a one-liner:
```
scheduled = if (date = article?.scheduled_publish_at) then moment(date).toISOString() else null
```